### ISSUE TITLE
GGMAX Bug - Level Editor Graphical Distortion

### ISSUE THREAD

**Issue by ZD-ZMA** [TheGameCreators/GameGuruRepo#5790]
i didnt make any changes to the level. its a brand new level and this is what happened. it shouldnt be like this obviously. i recorded a video of the bug as well as my specs.

https://youtu.be/18OzEa547sU

[https://youtu.be/18OzEa547sU](url)

**Comment by ZD-ZMA** [comment]
this happens on all of the terrain types. i just found out

**Comment by Kasseyus** [comment]
It looks like GPU was maxed out during that demonstration.  

The video suggested you are using a Radeon RX 6500 XT.  Is this so?  Also, you also mentioned an update...are you running the latest drivers?

**Comment by ZD-ZMA** [comment]
> It looks like GPU was maxed out during that demonstration.
> 
> The video suggested you are using a Radeon RX 6500 XT. Is this so? Also, you also mentioned an update...are you running the latest drivers?

yes to both

**Comment by ZD-ZMA** [comment]
> It looks like GPU was maxed out during that demonstration.
> 
> The video suggested you are using a Radeon RX 6500 XT. Is this so? Also, you also mentioned an update...are you running the latest drivers?

also nothing else was running in the background. nothing was running or updating at the time. (Obviously i recorded it so i had that running, but it was happening even when i wasn't recording.)

**Comment by davetgc** [comment]
Does this also happen in test game?

Could you also add your full system specs please?

**Comment by Kasseyus** [comment]
@davetgc According to the video, it's Ryzen 5600 6 Core, a Radeon RX 6500 XT and 16GB RAM 

@ZD-ZMA 
1> Does this happen every time you run MAX?
2> If so, can you run another video test, but this time in 'Test Game' mode and see if the same issue with the GPU load occurs?

**Comment by ZD-ZMA** [comment]
> Does this also happen in test game?
> 
> Could you also add your full system specs please?

no, not in test game. only in the editor. also i can try and do another full system specs again, but ill have to try and remember how since its been awhile since i last posted my specs on github and i dont normally look it up, sorry

**Comment by ZD-ZMA** [comment]
> @davetgc According to the video, it's Ryzen 5600 6 Core, a Radeon RX 6500 XT and 16GB RAM
> 
> @ZD-ZMA 1> Does this happen every time you run MAX? 2> If so, can you run another video test, but this time in 'Test Game' mode and see if the same issue with the GPU load occurs?

yes everytime however i already know it doesnt happen in test mode. its just specifically editor mode. i also want to point out it doesnt happen in the terrain generator mode either... just the editor only.

**Comment by ZD-ZMA** [comment]
Device name	DESKTOP-COUEN9F
Processor	AMD Ryzen 5 5600 6-Core Processor                 3.50 GHz
Installed RAM	16.0 GB
Device ID	40D520CC-03D0-4BFB-8A89-385BD609522D
Product ID	00330-80000-00000-AA757
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

Edition	Windows 10 Pro
Version	22H2
Installed on	‎12/‎26/‎2023
OS build	19045.5011
Experience	Windows Feature Experience Pack 1000.19060.1000.0

i copy pasted everything. i hope this answers what you were asking in regards to FULL system specs. if im missing something, let me know

**Comment by Kasseyus** [comment]
@ZD-ZMA So, it seems that this issue is only present on the terrain generator screen, correct?

Can you please test something for me.  In that example linked above, it was a forested biome, so lots of trees, plants, etc at an enormous height, so your graphics card had to do a lot of work.

Can you try the same experiment, but with a less populated biome, such as desert?  Same actions, and check the hardware status like last time, and see what the effect on the graphics card is. 

It would be interesting to see if your card has to work just as hard with less stuff to draw...

**Comment by ZD-ZMA** [comment]
I've already done so. Happens even with 0 trees

**Comment by LeeBamberTGC** [comment]
NOTE: Just going through the older 2024 bug reports marked WAITING FOR MORE INFO.  Feel free to add new info here using the latest DEVTEST build. If nothing new is posted, I will assume it cannot be reproduced with the newer updates and close this one in the near future.

**Comment by ZD-ZMA** [comment]
this is no longer an issue. this was fixed.
